slasher/beaconclient/receivers.go

```diff
@@ -94,7 +100,9 @@ func (bs *Service) collectReceivedAttestations(ctx context.Context) {
 				log.WithError(err).Error("Could not save indexed attestation")
 				continue
 			}
-			log.Infof("%d attestations for slot %d saved to slasher DB", len(collectedAtts), collectedAtts[0].Data.Slot)
+			log.WithFields(logrus.Fields{
+				"slot": collectedAtts[0].Data.Slot,
+			}).Infof("%d attestations saved to slasher DB", len(collectedAtts))
```

Comment:
```suggestion
			}).Info("attestations saved to slasher DB")
```
Turn `len(collectedAtts)` into a field. Or else it defeats the purpose of using `log.WithFields` as turning it away from a generalized log

Reply:
+1 @0xKiwi

Reply:
Thought about doing that but was confused on how to format it, how is it now?